src/EA.Weee.Web/Areas/Admin/Controllers/AatfController.cs

```diff
@@ -32,6 +33,36 @@ public AatfController(Func<IWeeeClient> apiClient, IWeeeCache cache, BreadcrumbS
         }
 
         [HttpGet]
+        public async Task<ActionResult> Details(Guid id)
+        {
```

Comment:
Missing controller tests

Reply:
Added some tests as well as mapping the view model with tests